.github/workflows/pre-merge-checks.yml

```diff
@@ -22,6 +23,23 @@ jobs:
       - name: Run Spotless
         run: ./gradlew --no-daemon spotlessCheck
 
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-function:
+          - get-addresses
```

Comment:
Strategy: Matrix: allows you to set variables that are then reused in the steps. 
The steps will run for same number of strategy that exist. In this use we would add a new node-function value whenever we add a new function.